### ISSUE TITLE
[Bug] Browser window not closed - MAUI Android

### ISSUE THREAD

**Issue by Pmr-precure** [AzureAD/microsoft-authentication-library-for-dotnet#4699]
### Library version used

4.60.1

### .NET version

MAUI .net 8

### Scenario

PublicClient - mobile app

### Is this a new or an existing app?

This is a new app or experiment

### Issue description and reproduction steps

After a successful login the browser keeps running with the login screen on.

I tested it my own project and in the MAUI samples project, same happens.

We have a Xamarin app that we are moving to MAUI, and here the browser closes after login.

![image](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/assets/99671597/016a6db2-5a8c-4e4b-b804-b67e4e364f3c)

### Expected behavior

Browser/Tab should close after login

### Identity provider

Azure B2C Custom Policy

### Solution and workarounds

No work arounds found.

**Comment by localden** [comment]
@Pmr-precure - and you're using the system browser, I am assuming?

**Comment by Pmr-precure** [comment]
Yes and tested it on multiple devices, both running Android 13 though.

**Comment by Pmr-precure** [comment]
No one else has this issue?

**Comment by localden** [comment]
@Pmr-precure - I will need to test this to verify the scenario. Thank you for the feedback, I will follow-up on this issue.

**Comment by Pmr-precure** [comment]
> @Pmr-precure - I will need to test this to verify the scenario. Thank you for the feedback, I will follow-up on this issue.

Did you have time to look at it? :) we are still facing this issue. BR

**Comment by localden** [comment]
@Pmr-precure also to confirm - this only happens for Azure B2C and not for Entra ID?

**Comment by Pmr-precure** [comment]
> @Pmr-precure also to confirm - this only happens for Azure B2C and not for Entra ID?

To be honest im not sure about the difference. We use a Azure b2c sign in only policy. Not custom. It works fine in our xamarin app. But in Maui it doesnt close the browser window after a login. BR

**Comment by bgavrilMS** [comment]
There is an AuthenticationContinuationHelper class which is supposed to be notified of when the browser flow finishes. It's in your code. Does this get hit at all?

**Comment by Pmr-precure** [comment]
> There is an AuthenticationContinuationHelper class which is supposed to be notified of when the browser flow finishes. It's in your code. Does this get hit at all?

It is in my OnActivityResult, but OnAcitivityResult doesnt trigger. I will try to do more testing.

**Comment by Pmr-precure** [comment]
> There is an AuthenticationContinuationHelper class which is supposed to be notified of when the browser flow finishes. It's in your code. Does this get hit at all?

Okay i just tested on my Pixel 8, here the OnActivityResult doesnt trigger at all. Then i tried on an Emulator, and it does trigger and call AuthenticationContinuationHelper. But it still doesnt close the brower window.

**Comment by Pmr-precure** [comment]
Finally break through!
I was missing this: 
![image](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/assets/99671597/038fa966-fcb6-4d2b-82f2-5f36bc558567)
In my manifest

**Comment by bgavrilMS** [comment]
Thx for following up @Pmr-precure !
